Components/HeroSlider/index.twig

```diff
@@ -0,0 +1,44 @@
+<div class="flyntComponent themeDark" is="flynt-hero-slider">
+  <script type="application/json">{{ jsonData|json_encode }}</script>
+  <div data-slider class="slider swiper-container">
+    <div class="swiper-wrapper">
+      {% for slide in slides %}
+      <div class="swiper-slide">
+        <picture class="content-picture">
+          <source srcset="
+            {{ slide.images.imageDesktop.src | resize(2560, 800, 'center') }} 2560w,
```

Comment:
I believe we should use `dynamicResize` here (and everywhere).

Reply:
done